lib/jekyll-seo-tag/json_ld_drop.rb

```diff
@@ -36,10 +38,13 @@ def fallback_data
       end
 
       def author
+        author_type = page_drop.author["type"]
```

Comment:
I prefer having this variable assigned after the check for `author["name"]` so that the variable isn't assigned unnecessarily.